### PR TITLE
Issue#328-Gear split into CurrentGear and SelectedGear.

### PR DESCRIPTION
**Pull request by UlfBj**

### CODE REVIEW

File: spec/Powertrain/Transmission.vspec [hunk 1]
```diff
@@ -70,12 +70,19 @@ TravelledDistance:
 #
 # Current gear
 #
-Gear:
+CurrentGear:
```
Comment (erikbosch) [inline spec/Powertrain/Transmission.vspec:73]: Here we touch the "deprecated" topic. If we know or can assume that no-one is using the old signal, then I think it is fine that we just remove/rename the existing signal. If not we should keep the old signal as well and mark it as deprecated, to give users time to change to the new signal name

**Comment by UlfBj** [comment]
GearSelect might be a better name than SelectedGear as it is an actuator.

File: spec/Powertrain/Transmission.vspec [hunk 2]
```diff
@@ -70,12 +70,19 @@ TravelledDistance:
...
+  datatype: int8
+  type: sensor
+  description: The current gear. 0=Neutral, 1/2/..=Forward, -1/..=Reverse
+
+
+#
+# Selected gear
+#
+SelectedGear:
   datatype: int8
```
Comment (danielwilms) [inline spec/Powertrain/Transmission.vspec:83]: here we could work with an enum, maybe?

**Comment by SebastianSchildt** [comment]
Generally, I approve the split

File: spec/Powertrain/Transmission.vspec (hunk 2 above)
Comment (danielwilms) [reply spec/Powertrain/Transmission.vspec:83]: meeting conclusion: We can leave it as is, but use deprecation.

**Pull request merged**